raphtory-graphql/src/model/plugins/mutation.rs

```diff
@@ -0,0 +1,59 @@
+use crate::model::plugins::mutation_entry_point::MutationEntryPoint;
+use async_graphql::{
+    dynamic::{Field, FieldFuture, FieldValue, InputValue, Object, ResolverContext, TypeRef},
+    FieldResult,
+};
+use dynamic_graphql::internal::{Register, Registry};
+use futures_util::future::BoxFuture;
+use crate::model::plugins::mutation_plugins::MutationPlugins;
+
+pub trait Mutation<'a, A: MutationEntryPoint<'a> + 'static> {
```

Comment:
this trait looks the same as the Query trait. Can't we have a Function trait with an apply function that can be used for both mutations and queries?